src/NHibernate/Linq/Visitors/LeftJoinRewriter.cs

```diff
@@ -57,14 +58,14 @@ private static void InsertBodyClauses(IEnumerable<IBodyClause> bodyClauses, Quer
 			foreach (var bodyClause in bodyClauses)
 			{
 				destinationQueryModel.BodyClauses.Insert(destinationIndex, bodyClause);
-				++destinationIndex;
+				destinationIndex++;
```

Comment:
Readability. I have seen some debate about performances of the former vs the later. Not only I do know believe it to be relevant for .Net (as Alexander it seems), but also I consider that such tiny optimization should by default be discarded in favor of readability. (Only in very specific cases where it can actually have a human measurable impact would I left such a thing.)